internal/service/encryptionatrestprivateendpoint/resource_migration_test.go

```diff
@@ -0,0 +1,13 @@
+package encryptionatrestprivateendpoint_test
+
+import (
+	"testing"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/mig"
+)
+
+func TestMigEncryptionAtRestPrivateEndpoint_basic(t *testing.T) {
+	mig.SkipIfVersionBelow(t, "1.18.1")
```

Comment:
1.19?

Reply:
Actually not certain what version this feature will target, @maastha do you have more context?

Reply:
synced offline, for now leaving at 1.19.0